rmw_connext_shared_cpp/src/shared_functions.cpp

```diff
@@ -346,6 +347,103 @@ create_node(
     return NULL;
   }
 
+  if (security_options->security_root_path) {
```

Comment:
done in https://github.com/ros2/rmw_connext/pull/225/commits/dda0cd0b30fa03eb08e5e23ef4d7d6b5a4560198